datafusion/spark/src/function/datetime/mod.rs

```diff
@@ -49,10 +51,15 @@ pub mod expr_fn {
         "Returns the last day of the month which the date belongs to.",
         arg1
     ));
+    export_functions!((
+        make_dt_interval,
+        "Make dt interval from weeks, days, hours, mins and secs.",
+        arg1 arg2 arg3 arg4 arg5
```

Comment:
```suggestion
        days hours mins secs
```

I think only four arguments right?